### PR TITLE
Update error message when lessee creates illegal sublease/suboffer

### PR DESCRIPTION
**Pull request by tzumainn**

### CODE REVIEW

**Review by ljmcgann — APPROVED** [review]
short and sweet

**Review by samuel-zuk — APPROVED** [review]
Looks good!

File: esileapclient/tests/functional/v1/test_time.py [hunk 1]
```diff
@@ -395,7 +395,9 @@ def test_offer_create_sublease_out_of_range(self):
 
         # NOTE: This error should be handled differently. When the way this
```
Comment (samuel-zuk) [inline esileapclient/tests/functional/v1/test_time.py:396]: Oh wait, I missed this. This comment should be deleted now.

Comment (tzumainn) [reply esileapclient/tests/functional/v1/test_time.py]: Oh right! Removed it now.

**Pull request merged**